### ISSUE TITLE
[New Dataset]: hyoununiversalstructure

### ISSUE THREAD

**Issue by LinguList** [lexibank/lexibank#158]
Data finally out now, can be found here: http://hyoun.me/language/reference.html

The underlying data is in [this json file](http://hyoun.me/language/res2.json). 

Adding this would require to identify the concepts, link them to concepticon, and fill in slots for the words per concept. Maybe a good usecase for cldf, could also be shown as a demo in a [blogpost](https://calc.hypotheses.org)?

**Comment by SimonGreenhill** [comment]
There's also a table here: http://hyoun.me/language/data/PST80.txt

**Comment by LinguList** [comment]
Yep. We'd need a taker to look at that, it seems to be pretty straightforward, @tresoldi, would you have time to have a look, or could you schedule it?
